NOTES.md

```diff
@@ -1,3 +1,19 @@
+# Release 2.1.4 (Forbidden Planet)
+
+Released on 2015-10-15
+
+Bug fix release
+
+* Fixed a second data corruption bug in the b-tree implementation (#4769)
```

Comment:
Why is it important to mention that it's a second bug?

B-tree should be capitalized.